google-cloud-tools-plugin/ultimate/src/com/google/cloud/tools/intellij/appengine/server/instance/AppEngineServerModel.java

```diff
@@ -47,24 +49,33 @@
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
 
+import java.io.File;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
 import java.util.Collections;
 import java.util.List;
 
 /**
  * @author nik
  */
-public class AppEngineServerModel implements ServerModel, DeploysArtifactsOnStartupOnly {
+public class AppEngineServerModel implements ServerModel, DeploysArtifactsOnStartupOnly,
+    RunConfiguration, Cloneable {
 
-  private ArtifactPointer myArtifactPointer;
-  private int myPort = 8080;
-  private String myServerParameters = "";
-  private CommonModel myCommonModel;
+  private ArtifactPointer artifactPointer;
+  private CommonModel commonModel;
+
+  private AppEngineModelSettings settings = new AppEngineModelSettings();
```

Comment:
just wondering why this Settings object needs to exist. Couldn't you just put all the fields directly on the AppEngineServerModel?